src/pages/messages/MyMessagesPage.tsx

```diff
@@ -22,11 +23,13 @@ export const MyMessagesPage = () => {
       <Helmet>
         <title>{t('my_page.messages.messages')}</title>
       </Helmet>
-      {isLoadingTicketsCollection ? (
-        <ListSkeleton minWidth={100} maxWidth={100} height={100} />
-      ) : (
-        <TicketAccordionList tickets={tickets} />
-      )}
+      <BackgroundDiv>
```

Comment:
BackgroundDiv kunne vært lagt inn i den øverste `<>` for å unngå en ekstra indentation